scripts/root-ca-retriever/dod-certs.ts

```diff
@@ -0,0 +1,175 @@
+/**
+ * Copyright 2025 Defense Unicorns
+ * SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+ */
+
+import * as https from "https";
+import * as fs from "fs";
+import * as path from "path";
+import * as crypto from "crypto";
+import AdmZip from "adm-zip";
```

Comment:
Curious if you evaluated other possible unzip tooling here - did a quick scan and there doesn't seem to be anything too "official" but there's a few that look maybe similar/better maintained?
- https://www.npmjs.com/package/unzipper
- https://www.npmjs.com/package/yauzl

Nothing seems particularly abandoned or unused, so it might be a coinflip here...